magento/magento2
# Checked if quote object contains id before looking for quote items

### Description (*)

Checked if quote object contains id before looking for quote items so that it doesn’t pull all the quote items
### Fixed Issues (if relevant)

1. magento/magento2#26437: Viewing customer shopping cart in admin shows all products in catalog when there is no active quote

### Manual testing scenarios (*)

1. Create a new customer account in storefront
2. Login to admin panel and go to Customers > All Customers path
3. Edit newly created customer and go to "Shopping Cart" tab
4. You will observe that it lists many items but the shopping cart of the customer in the storefront is blank
5. But if you don't see all items here, go to database table "quote" & search it for customer id, if any quote found delete that
6. Now, refresh the customer edit page in backend & go to "Shopping Cart" tab again

### Questions or comments


### Contribution checklist (*)
